hi-ml-cpath/src/health_cpath/configs/classification/BaseMIL.py

```diff
@@ -316,27 +317,11 @@ def create_model(self) -> TilesDeepMILModule:
         return deepmil_module
 
 
-class BaseMILSlides(BaseMIL):
+class BaseMILSlides(BaseMIL, TilingParams):
     """BaseSlidesMIL is an abstract subclass of BaseMIL for running MIL experiments on slides datasets. It is
     responsible for instantiating the full DeepMIL model in slides settings. Subclasses should define their datamodules
     and configure experiment-specific parameters.
     """
-    # Slides Data module parameters:
-    tile_size: int = param.Integer(224, bounds=(0, None), doc="Size of the square tile, defaults to 224.")
-    step: int = param.Integer(None, bounds=(0, None),
-                              doc="Step size to define the offset between tiles."
-                              "If None (default), it takes the same value as tile_size."
-                              "If step < tile_size, it creates overlapping tiles."
```

Comment:
I refactored this into TilingParams to avoid having to copy parameters+docs at multiple levels: Datamodule/base and sub containers...

Reply:
that made the code a lot nicer, thank you!